Model/ThirdPartyModuleFactory.php

```diff
@@ -77,7 +77,17 @@ public function isAvailable()
      */
     public function isExists()
```

Comment:
@dumega please remind me why do we need isExists.

Reply:
Since the Amasty team completely overhauled Amasty GiftCard module, module is enabled does not mean the specific class exists, instead of comparing the version, I think checking the class existence is more reliable.